### DISCUSSION TITLE
PROSE Powerfx SDK?

### DISCUSSION THREAD

**Discussion by BugBareDrums** [microsoft/Power-Fx#2401] [General]
Hi,
I am planning to use PowerFx to enable users with excel knowledge to write expressions in a C# app. I would like to allow them to generate these expressions with AI. I know PROSE can generate PowerFx inside of Excel, are there any plans to make this feature available in custom apps? As an API or SDK perhaps?

thanks
Nick

**Comment by MikeStall** [comment]
Yes :) This is underway and we will expose the NL2Fx functionality publicly.

**Reply by rajyraman** [reply]
@MikeStall - Great news!!

**Reply by BugBareDrums** [reply]
Thanks @MikeStall ☺️
